spec.html

```diff
@@ -36297,16 +36297,24 @@ <h1>
           <h1>
             GetStringIndex (
               _S_: a String,
-              _e_: a non-negative integer,
+              _cpIndex_: a non-negative integer,
             ): a non-negative integer
           </h1>
           <dl class="header">
+            <dt>description</dt>
+            <dd>It interprets _S_ as a sequence of UTF-16 encoded code points, as described in <emu-xref href="#sec-ecmascript-language-types-string-type"></emu-xref>, and returns the code unit index corresponding to code point index _cpIndex_ when such an index exists. Otherwise, it returns the length of _S_.</dd>
           </dl>
           <emu-alg>
             1. If _S_ is the empty String, return 0.
-            1. Let _codepoints_ be StringToCodePoints(_S_).
-            1. Let _eUTF_ be the smallest index into _S_ that corresponds to the character at element _e_ of _codepoints_. If _e_ is greater than or equal to the number of elements in _codepoints_, then _eUTF_ is the length of _S_.
-            1. Return _eUTF_.
+            1. Let _len_ be the length of _S_.
+            1. Let _cuCount_ be 0.
+            1. Let _cpCount_ be 0.
```

Comment:
Please spell out `_cuCount_`, `_cpCount_`, and `_cpIndex_` to `_codeUnitCount_`, `_codePointCount_`, and `_codePointIndex_`. An edit distance of 1 letter increases likelihood of typos, bugs, etc.

Reply:
I thought so too, but these seemed to be the only `c[up]…` names so I just updated as suggested.

Reply:
No, there's a lot. `Canonicalize` and `UTF16EncodeCodePoint` are the only I see that have *both* `cp` and `cu` aliases, but plenty of aliases are abbreviated `cp` and `cu` throughout the spec. It's fine to make the requested change here, but if @syg wants to avoid this potential for error, we should open a separate issue to perform a more general renaming.